src/models/User.js

```diff
@@ -1,22 +1,66 @@
 export default (sequelize, DataTypes) => {
-  const User = sequelize.define('User', {
-    firstName: DataTypes.STRING,
-    lastName: DataTypes.STRING,
-    email: DataTypes.STRING,
-    password: DataTypes.STRING,
-  }, {
-    classMethods: {
-      associate: (models) => {
-        User.hasMany(models.Requests, {
-          onDelete: 'CASCADE',
-          foreignKey: 'UserId'
-        });
-        User.hasOne(models.Accomodations, {
-          foreignKey: 'UserId',
-          onDelete: 'CASCADE'
-        });
+  const User = sequelize.define(
+    'User',
+    {
+      firstName: DataTypes.STRING,
+      lastName: DataTypes.STRING,
+      email: {
+        type: DataTypes.STRING,
+        unique: true,
+        allowNull: false,
+        validate: {
+          isEmail: {
+            args: true,
+            msg: 'Email address is invalid.'
+          },
+          notNull: {
+            args: true,
+            msg: 'Email address field cannot be empty.'
```

Comment:
Good job @ChidozieGabriel. However, I feel `Email address cannot be empty.` is more suitable.

Reply:
I've resolved it. Thanks.